metals/src/main/scala/scala/meta/internal/metals/testProvider/frameworks/ScalatestTestFinder.scala

```diff
@@ -320,4 +356,18 @@ object ScalatestStyle {
     override val leafMethods: Set[String] = Set("in", "ignore")
   }
 
+  case object AnyFeatureSpec extends ScalatestStyle {
+    val symbols: Set[String] = Set(
+      "org/scalatest/freespec/AnyFeatureSpec#",
+      "org/scalatest/freespec/AnyFeatureSpecLike#",
+      "org/scalatest/freespec/AsyncFeatureSpec#",
+      "org/scalatest/freespec/AsyncFeatureSpecLike#",
+      "org/scalatest/freespec/FixtureAnyFeatureSpec#",
+      "org/scalatest/freespec/FixtureAnyFeatureSpecLike#",
+      "org/scalatest/freespec/FixtureAsyncFeatureSpec#",
+      "org/scalatest/freespec/FixtureAsyncFeatureSpecLike#",
```

Comment:
```suggestion
      "org/scalatest/featurespec/AnyFeatureSpec#",
      "org/scalatest/featurespec/AnyFeatureSpecLike#",
      "org/scalatest/featurespec/AsyncFeatureSpec#",
      "org/scalatest/featurespec/AsyncFeatureSpecLike#",
      "org/scalatest/featurespec/FixtureAnyFeatureSpec#",
      "org/scalatest/featurespec/FixtureAnyFeatureSpecLike#",
      "org/scalatest/featurespec/FixtureAsyncFeatureSpec#",
      "org/scalatest/featurespec/FixtureAsyncFeatureSpecLike#",
```

tested it locally and this works, you also need to add the new style to the list of styles in this file.

It also looks like there are some issues running those test and it can't run the added cases.

I would try to add some tests that actually run the suite maybe or at least testing it manually. I am guessing we are sending a wrong test name to the test framework.

Reply:
Oooh my bad. I will take a look!